### PR TITLE
Initial commit on Twitter Integrations

### PR DESCRIPTION
**Pull request by Prashanth-TR**
Please review the Twitter project I have done as a part of the bootcamp process.

### CODE REVIEW

File: src/main/java/com/Bootcamp/TwitterBasic/BasicTwitterFeatures.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class BasicTwitterFeatures {
+    public void postTweet(String text)
+    {
+        Twitter twitter = TwitterFactory.getSingleton();
```
Comment (NarendraPrabhu) [inline src/main/java/com/Bootcamp/TwitterBasic/BasicTwitterFeatures.java:14]: While it is a singleton, it is alright to call it any number of times for each function, just make sure TwitterrFactory.getSingleton() isn't time or memory consuming.

Also good time to read and know about Singleton design pattern.

File: src/main/resources/twitter4j.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+oauth.consumerKey = ******
```
Comment (NarendraPrabhu) [inline src/main/resources/twitter4j.properties:1]: Good one.

**Review by NarendraPrabhu — APPROVED** [review]
Very good one, try expanding features into different classes, Read through OO Class Design principles and and SOLID rules, try adopting them. Also try expanding timeline feeds and post feature.

**Comment by Prashanth-TR** [comment]
Hi Narendra,
Thank you for your response. Will explore on the suggestions you have
provided and clarify with you tomorrow if there are any doubts.

Regards,
Prashanth TR

**Comment by NarendraPrabhu** [comment]
Sure Sir

**Pull request merged**